site-scrapers/LowellGeneral/index.js

```diff
@@ -141,7 +141,8 @@ function parseAvailability(body) {
                     ].innerText.replace(/ {2,}/g, " ");
                     const thMonth = headerText.split(" ")[1];
                     const thDay = leadZero(headerText.split(" ")[2]);
-                    const [month, year] = getYearForMonth(thMonth);
+                    let [month, year] = getYearForMonth(thMonth);
```

Comment:
https://momentjs.com/docs/#/displaying/format/
Have you thought about using moment for this? MMM = the three-day short month string, so you wouldn't have to hard-code that, and you can also parse it and then spit it back out in the format you want (with padded 0s). LMK what you think.

Reply:
Good call. Uploaded a fix for this using moment.